2.4.41testenv/test/run

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# 
+# Test the OpenLDAP test-env image.
+#
+# IMAGE_NAME specifies the name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
```

Comment:
maybe a comment in here about why this is empty?  in general i'm a little puzzled by this "2.4.41testenv" image..what's it for?